spec/features/user_votes_on_reviews_spec.rb

```diff
@@ -3,30 +3,84 @@
 require 'rails_helper'
 
 feature 'User votes on reviews' do
-  let!(:user) { FactoryGirl.create(:user) }
-  let!(:recipe) { FactoryGirl.create(:recipe) }
   let!(:review) { FactoryGirl.create(:review) }
+  let!(:user) { review.user }
+  let!(:recipe) { review.recipe }
+  let!(:user2) { FactoryGirl.create(:user, first_name: "Jane") }
 
   context 'As a user' do
+    scenario 'I can see the upvotes and downvotes' do
+      visit recipe_path(recipe)
+
+      expect(page).to have_content '0'
+    end
+
+    scenario 'I cannot upvote a review if I\'m not signed in.' do
+      visit recipe_path(recipe)
+      click_button 'upvote'
+
+      expect(page).to have_content 'Please sign in'
+    end
+
+    scenario 'I cannot downvote a review if I\'m not signed in.' do
+      visit recipe_path(recipe)
+      click_button 'downvote'
+
+      expect(page).to have_content 'Please sign in'
+    end
+
     scenario 'I can upvote a review' do
       user_sign_in(user)
+      visit recipe_path(recipe)
+      click_button'upvote'
+
+      expect(review.votes[0].value).to eq 1
     end
 
     scenario 'I can downvote a review' do
+      user_sign_in(user)
+      visit recipe_path(recipe)
+      click_button 'downvote'
+
+      expect(review.votes[0].value).to eq -1
```

Comment:
This is one case where I sorta-kinda agree with hound. 

Due to ruby magics, it is not quite sure whether you mean a negative number, or you mean to somehow subtract 1 from `expect(review.votes[0].value).to eq`. Since the latter is nonsense, it might be a good idea to wrap these in parenthesis moving forward.